ogc-example/src/main/resources/application.conf

```diff
@@ -369,4 +375,21 @@ color-maps = {
     2500: 0xD7D7D7FF,
     3000: 0xFFFFFFFF
   }
+  "ndvi": {
+    -1.0: 0x1947B0FF,
```

Comment:
btw, is not it better just to have strings everywhere if doubles are used? just for simplicity?

Reply:
Whose simplicity? I'm open to coming down on a best practice hard if it is obviously useful but am concerned that hard+fast rules about quoting in some contexts rather than others will make user's lives a bit more difficult (as, generally, HOCON seems to be aimed at a very permissive syntax)